### ISSUE TITLE
A run command

### ISSUE THREAD

**Issue by KotlinIsland** [KotlinIsland/basedmypy#528]
```py
print("hi")
```console
> mypy --run test.py
Success: no issues found in 1 source file
hi
```
